### PR TITLE
ISSUE: Show form and Submit answers

### PR DESCRIPTION
**Pull request by yohanology**
I'm stuck to submit the answers. The answers are successfully submitted but the values of each is the last value of the form.

### CODE REVIEW

**Comment by yohanology** [comment]
I fixed the problem, could you check it please. thx

xoxo

File: www/js/controllers.js [hunk 1]
```diff
@@ -54,6 +54,59 @@ angular.module('starter.controllers', [])
...
+          values: answerValues
+        }
+        console.log("answer", answer)
+        
+        $http.post(apiUrl + '/contents/' + content.id + '/answers', answer).success(function(response) {
+          console.log("response", response);
+        });
+      });
+
+      $ionicPopup.alert({
```
Comment (h8rry) [inline www/js/controllers.js:93]: This will be called even when not all answers have been submitted.

Comment (h8rry) [inline www/js/controllers.js:93]: Notice the $http ajax request inside the for loop.

Comment (h8rry) [inline www/js/controllers.js:93]: It's better to just send 1 request, and do all the other things inside your Rails controller

**Comment by h8rry** [comment]
@yohanology please review

File: www/js/controllers.js (hunk 1 above)
Comment (ctlong) [inline www/js/controllers.js:93]: We have to submit an answer for each content, so how would we do it without a for loop?

Comment (h8rry) [inline www/js/controllers.js:93]: Submit all answers as a hash. Process inside Rails.

**Comment by yohanology** [comment]
Fixed. Please merge!

File: www/js/controllers.js [hunk 2]
```diff
@@ -54,6 +54,57 @@ angular.module('starter.controllers', [])
...
+      
+      var answers = {};
+      
+      submissionId = resp.id;
+      $scope.contents.forEach(function(content) {
+        answerValues = [$scope.submits[content.index]];
+        answers[content.id] = answerValues;
+        
+      });
+        console.log("answers", answers)
```
Comment (h8rry) [inline www/js/controllers.js:87]: Indentation

**Comment by h8rry** [comment]
Can merge after fixing indentation

**Pull request merged**